.github/run_test.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
+export MACHINE_ID=1fdb4634-15de-4327-8b92-b86703b7012a
```

Comment:
This works due to me passing this. Seems like somehow peg has changed to not provide this or vboxmanage no longer accepts an empty uuid/name for the machine while it did before?

Any idea cc @jimmykarily ? I think you were involved in this part of the peg code?

Reply:
I can't find this change in the PR anymore. Is it ok now?

Reply:
Yep, just changed it to generate a random UUID like the rest of our projects do.